### PR TITLE
Update OpenGraph meta tags to use incident title and description

### LINKED ISSUE #871: Update OpenGraph meta tags to use incident title and description
On `cite.js`, change the `metaTitle` and `metaDescription` to start using the title and description from the incident itself.

### PR DESCRIPTION
**Pull request by pdcp1**
- On `cite.js`, change the `metaTitle` and `metaDescription` to start using the title and description from the incident itself.
- If the language is `en` use the actual incident title on the incident page:

### Before:

image

### After:

image

 

Closes #871

### CODE REVIEW

File: site/gatsby-site/src/templates/cite.js [hunk 1]
```diff
@@ -125,7 +132,7 @@ function CitePage(props) {
       </AiidHelmet>
 
       <div className={'titleWrapper'}>
-        <StyledHeading>{metaDescription}</StyledHeading>
+        <StyledHeading>{locale == 'en' ? metaTitle : defaultIncidentTitle}</StyledHeading>
```
Comment (lmcnulty) [inline site/gatsby-site/src/templates/cite.js:135]: I think Sean wanted it to have the incident ID *and* the title.

Comment (smcgregor) [reply site/gatsby-site/src/templates/cite.js:135]: Yes please!

Comment (pdcp1) [reply site/gatsby-site/src/templates/cite.js:135]: @smcgregor This way sounds good to you?
image

**Review by lmcnulty — APPROVED** [review]

File: site/gatsby-site/src/templates/cite.js (hunk 1 above)
Comment (smcgregor) [reply site/gatsby-site/src/templates/cite.js:135]: Looks good to me!

**Pull request merged**